templates/components/styles/crosslinks.html.twig

```diff
@@ -7,7 +7,7 @@
                     <p>{{ crosslinks['text'] }}</p>
                 {% endblock %}
             </div>
-            <ul class="clean-list" role="presentation">
+            <ul class="clean-list" role="presentation" data-anchor="no">
```

Comment:
@jean-gui This should not be needed here, because:

- the script only targets headings within `<main>`
- the parent container for this component is `<aside>` and in the templates I do not believe it is nested within `<main>`. It is not intended to be, as per the original designs.

Reply:
Good catch, I've remove it